federation/pkg/federation-controller/secret/secret_controller_test.go

```diff
@@ -183,13 +208,17 @@ func GetSecretFromChan(c chan runtime.Object) *api_v1.Secret {
 
 // Wait till the store is updated with latest secret.
 func WaitForSecretStoreUpdate(store util.FederatedReadOnlyStore, clusterName, key string, desiredSecret *api_v1.Secret, timeout time.Duration) error {
-	retryInterval := 100 * time.Millisecond
+	retryInterval := 200 * time.Millisecond
 	err := wait.PollImmediate(retryInterval, timeout, func() (bool, error) {
 		obj, found, err := store.GetByKey(clusterName, key)
 		if !found || err != nil {
+			glog.Errorf("%s is not in the store", key)
 			return false, err
 		}
 		equal := secretsEqual(*obj.(*api_v1.Secret), *desiredSecret)
+		if !equal {
+			glog.Errorf("wrong content in the store expected:\n%v\nactual:\n%v\n", *desiredSecret, *obj.(*api_v1.Secret))
```

Comment:
Well this is not an error. This is expected to fail until eventually it becomes equal.
Change this to Infof?